automation/tests-container-utils.sh

```diff
@@ -70,10 +70,6 @@ function container_pre_test_setup {
     set -x
     ${CONTAINER_CMD} --version && cat /etc/resolv.conf
 
-    if [[ "$CI" == "true" ]];then
-        rebuild_container_images
-    fi
```

Comment:
Can you explain in the commit message what is the reason not building the container image in the 1.4 branch ?

Reply:
Done.